tests/E2E/e2e_basiclinuxoffline_test.ps1

```diff
@@ -0,0 +1,77 @@
+#Requires -RunAsAdministrator
+function Setup-BasicLinuxNodeOffline {
```

Comment:
No Windows testing?

Reply:
I just wanted to setup a testing framework for the scripts with an initial set of test cases first, but to your point we should probably have test cases for windows as well.

Reply:
We could approve the PR with Linux only, but with the upcoming GA of Windows nodes, we should include those tests in the near future